docs/conf.py

```diff
@@ -11,7 +11,11 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
+
+# To set the following values to the latest version and
```

Comment:
I suggest you move this comment so that it sits just before (current) lines 31-34,  similar to the following:

```
# To set the following values to the latest version and
# release, run `python3 docs/version_release_fix.py`

# The short X.Y version
version = u'0.3.6'
# The full version, including alpha/beta/rc tags
release = u'0.3.6'
```

Otherwise it's might not be clear to the user what 'the following values' is referring to.

Reply:
I have made the changes. Please review.

Reply:
LGTM, thanks!